src/components/organisms/GlobalNav/GlobalNav.js

```diff
@@ -93,6 +94,8 @@ const GlobalNav = forwardRef((props, globalNavRef) => {
         <GlobalNavLinkHolder position={navDownPosition}>
           <Heading>{manualKerning("Bear meets eagle on fire")}</Heading>
         </GlobalNavLinkHolder>
```

Comment:
You will need to move these into the looping scroll component without there GlobalNavLinkHolder wrappers. The looping scroll component mimics the header and uses scrolltrigger to update the scroll as soon as it's top hits the top of the browser. 

This should solve seeing the page underneath the words as the Header and Heading components have a background colour on them. 

The loopingScoll component currently has a few thing in it that will need to be removed from the original build (the useHorizontalClassnames) for example. Go through and remove those. 

You may also need to wrap the looping scroll in a globalnavlink holder component to make sure it animates down when a menu item is clicked. 

FUN!

Reply:
Addressed below (I think I got everything,,)